type-definitions/immutable.d.ts

```diff
@@ -759,11 +759,15 @@ declare namespace Immutable {
    * but since Immutable Map keys can be of any type the argument to `get()` is
    * not altered.
    */
-  function Map<K, V>(collection?: Iterable<[K, V]>): Map<K, V>;
-  function Map<V>(obj: { [key: string]: V }): Map<string, V>;
-  function Map<K extends string | symbol, V>(obj: { [P in K]?: V }): Map<K, V>;
-
-  interface Map<K, V> extends Collection.Keyed<K, V> {
+  function Map<T extends Array<[string | number | symbol, unknown]>>(collection: T): Map<{
```

Comment:
You changed from `Iterable` to `Array`. Can you make it back to iterable ? A `List` is iterable, but not an Array for example (but I'm not that sure as there is a test that validate this case 🤔 )